engines/wasmtime/wasmtime.go

```diff
@@ -464,6 +494,16 @@ func (i *Instance) Close(context.Context) error {
 	return nil // wasmtime only closes via finalizer
 }
 
+// Unwrap allows access to wasmtime-specific features.
+func (m *Module) Unwrap() *wasmtime.Module {
+	return m.module
+}
+
+// UnwrapStore allows access to wasmtime-specific features.
+func (m *Module) UnwrapStore() *wasmtime.Store {
```

Comment:
maybe add functions like this to wazero for parity, noting that the naming suffix should reflect the type returned and wazero doesn't call things store :D

Reply:
Makes sense, will do.